.eslintrc.js

```diff
@@ -23,19 +42,41 @@ module.exports = {
     ],
     'linebreak-style': [
       'error',
-      (process.platform === 'win32' ? 'windows' : 'unix') // all windows platforms are denoted by win32
+      'unix'
```

Comment:
Make the default linebreak style as LF (unix-style), as this ternary configuration really broke on every text editor / IDE I use on WIndows.